explore-analyze/dashboards/open-dashboard.md

```diff
@@ -17,32 +17,25 @@ products:
    When looking for a specific dashboard, you can filter them by tag or by creator, or search the list based on their name and description. Note that the creator information is only available for dashboards created on or after version 8.14.
    ::::
 
-3. Click the dashboard **Title** you want to open.
-4. Make sure that you are in **Edit** mode to be able to make changes to the dashboard. You can switch between **Edit** and **View** modes from the toolbar.
-   ![Switch between Edit and View modes](https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt619b284e92c2be27/6750f3a512a5eae780936fe3/switch-to-view-mode-8.17.0.gif)
+3. Click the dashboard name you want to open.
+   The dashboard opens automatically in **Edit** mode. You can switch between edit and view modes from the toolbar.
```

Comment:
```suggestion
   The dashboard opens automatically in **Edit** mode if you have the right permissions. You can switch between edit and view modes from the toolbar.
```
It won't open in edit mode if the user doesn't have edit permissions on that dashboard